sdk/src/math/superStake.ts

```diff
@@ -336,23 +432,15 @@ export async function calculateSolEarned({
 			} else {
 				solEarned = solEarned.add(record.amount);
 			}
-		} else if (record.marketIndex === 2) {
-			const msolRatio = lstRatios.get(record.ts.toNumber());
-			const msolRatioBN = new BN(msolRatio * LAMPORTS_PER_SOL);
-
-			const solAmount = record.amount.mul(msolRatioBN).div(LAMPORTS_PRECISION);
-			if (isVariant(record.direction, 'deposit')) {
-				solEarned = solEarned.sub(solAmount);
-			} else {
-				solEarned = solEarned.add(solAmount);
-			}
-		} else if (record.marketIndex === 6) {
-			const jitoSolRatio = lstRatios.get(record.ts.toNumber());
-			const jitoSolRatioBN = new BN(jitoSolRatio * LAMPORTS_PER_SOL);
-
-			const solAmount = record.amount
-				.mul(jitoSolRatioBN)
-				.div(LAMPORTS_PRECISION);
+		} else if (
+			record.marketIndex === 2 ||
+			record.marketIndex === 6 ||
+			record.marketIndex === 8
```

Comment:
could we add a little util function like `tokenIsSolLST` or something just so this is a bit clearer?